src/components/SignInButtons/AppleSignIn/index.website.tsx

```diff
@@ -49,23 +41,22 @@ const config = {
  * Apple Sign In success and failure listeners.
  */
 
-const successListener = (event) => {
+const successListener = (event: AppleIDSignInOnSuccessEvent) => {
     const token = event.detail.authorization.id_token;
     Session.beginAppleSignIn(token);
 };
 
-const failureListener = (event) => {
+const failureListener = (event: AppleIDSignInOnFailureEvent) => {
     if (!event.detail || event.detail.error === 'popup_closed_by_user') {
         return null;
     }
-    Log.warn(`Apple sign-in failed: ${event.detail}`);
+    Log.warn(`Apple sign-in failed: ${event.detail.error}`);
```

Comment:
could you please explain why the log is being changed?

```suggestion
    Log.warn(`Apple sign-in failed: ${event.detail.error}`);
```

Reply:
@rushatgabhane 
![Screenshot 2024-01-04 at 16 16 10](https://github.com/Expensify/App/assets/46921547/13e78dad-e838-4260-9ecb-5af3b258020a)

Reply:
@rushatgabhane 
![Screenshot 2024-01-04 at 16 17 57](https://github.com/Expensify/App/assets/46921547/3661410e-2720-4d9c-b6df-b6a9b3ab3d44)